backend/clubs/models.py

```diff
@@ -873,6 +873,7 @@ class Profile(models.Model):
 
     user = models.OneToOneField(get_user_model(), on_delete=models.CASCADE, primary_key=True)
     image = models.ImageField(upload_to=get_user_file_name, null=True, blank=True)
+    uuid_secret = models.UUIDField(default=uuid.uuid4, editable=False)
```

Comment:
Consider making this `editable`, you might want to provide a way for the user to regenerate their UUID secret if it is compromised.

Some tools, like Canvas, have this functionality available.